s3/src/main/scala/akka/stream/alpakka/s3/S3Settings.scala

```diff
@@ -67,7 +69,7 @@ object S3Settings {
   /**
    * Scala API: Creates [[S3Settings]] from a [[com.typesafe.config.Config Config]] object.
    */
-  def apply(config: Config, configurationPrefix: String = "akka.stream.alpakka.s3"): S3Settings = {
+  def apply(config: Config, configurationPrefix: String = configPath): S3Settings = {
```

Comment:
With this rewrite, the `config.getConfig` should be removed and always read directly from there.